.github/workflows/test.yaml

```diff
@@ -18,6 +32,7 @@ jobs:
           wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc 2>/dev/null | sudo apt-key add -
           sudo apt-get update
           sudo apt-get -y install postgresql-14
+          sudo apt-get -y install postgresql-16
```

Comment:
You can remove this line and the one above it. The `services` section of the yaml now makes sure this is being installed

Reply:
you can actually replace this whole `run:` with 
`run: pip install -r requirements.txt -r requirements-dev.txt`